data/map planets.txt

```diff
@@ -229,8 +232,8 @@ planet Asrayapur
 planet Swarajah
 	attributes "dirt belt" factory farming urban
 	landscape land/ncity7
-	description "Swarajah was one of the first colonies created by the Hindu peoples of Earth, becoming one of the more heavily populated planets in the south, rivaling Solace, however because of lack of infrastructure not much is industrialized so the surface is still filled with farms that feed into the densely packed cities."
-	spaceport "The spaceport is rather simplistic for such a large population, a simple concrete platform where large freighters come and go, further north you can spot the city it probably serves. Exploring the port you find an area where many merchants come to set up shop, thousands of people come to this market to buy and sell goods."
+	description "Swarajah was the first colony established by the Hindu peoples of Earth. It has become one of the more heavily populated planets in the South, but because of lack of infrastructure not much is industrialized as other planets in the region like Solace, so the surface is still filled with farms that feed the densely packed cities."
+	spaceport "The spaceport is rather simplistic for such a large population, just a series of simple concrete platforms where large freighters come and go. Further north you can spot the city it serves. Nearby there's an area especially designated as a marketplace for the local merchants, but it seems to be overflowing into the spaceport; there must be upwards of ten thousand people at this time, buying or selling goods."
```

Comment:
Swarajah will be the second most populous planet after solace with 1.4 billion. im planning a mission chain to move the capital of the FW to here or do a split like i did with the Republic.

Reply:
I think it would be more interesting if you left the split, `#1` because it would be more interesting, `#2` because it emphasises the differences between the Free Worlds and the Republic (FW propaganda might be something like, "Look at the Republic's government and how inefficient they are! They have three capitals, all their politicians have to shuttle around everywhere all the time! We keep everything on one planet and look how much more efficient we get!" Even if it might not be true.

Reply:
Also, consider whether the Rim's relative technological superiority would affect this. Southbound and Kraz are still major players, Solace is more aligned with the Rim, and Bourne is a major industrial centre.